api/aladin/views.py

```diff
@@ -0,0 +1,23 @@
+import django_filters
+from rest_framework import viewsets
+from rest_framework import filters
+from .models import Image
+from .serializers import ImageSerializer
+
+
+class ImageFilter(django_filters.FilterSet):
```

Comment:
Ok, documentei as duas formas

https://github.com/linea-it/dri/wiki/Creating-a-new-app#new-view